.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Check
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
On push/pull_request is good, but seems most value would come from a regular nightly job (did the binary packages or container images break the action?).

Reply:
Added a cron job. This has the same issue as any other repo with scheduled jobs in that somebody would actually need to track it.